tools/run_tests/helper_scripts/build_python.sh

```diff
@@ -157,11 +157,11 @@ curl https://bootstrap.pypa.io/get-pip.py | $VENV_PYTHON
 # pip-installs the directory specified. Used because on MSYS the vanilla Windows
 # Python gets confused when parsing paths.
 pip_install_dir() {
-  PWD=$(pwd)
+  PREV_PWD=$(pwd)
```

Comment:
Good catch! Although, looking at this pattern, I can't help but feel that we should be using [`pushd` and `popd`](https://www.gnu.org/software/bash/manual/html_node/Directory-Stack-Builtins.html).

Reply:
Indeed, I updated the branch to use `pushd` and `popd` instead. Thanks for the suggestion!